web/src/app/projects/create-edit/create-edit.component.html

```diff
@@ -1,8 +1,8 @@
 <div class="mat-content" fxLayout="column" fxLayoutGap="20px">
-    <form (submit)="save()" [formGroup]="projectForm">
+    <form (submit)="create()" [formGroup]="projectForm">
```

Comment:
Because the form is for `create` and `edit`, I think the naming `save` would be more appropriate

Reply:
Review incorporated. 👍